src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import './App.scss';
 
+import Sum from './components/Sum';
+
 const App = () => (
-  <h1>React sum</h1>
+  <div className="App">
+    <h1>React sum</h1>
+    <Sum
+      valueA={2}
+      valueB={2}
+    />
```

Comment:
если помещается в 1 строку, не нужно разделять на 4
```suggestion
    <Sum valueA={2} valueB={2} />
```